nautobot/dcim/filters/__init__.py

```diff
@@ -392,6 +398,26 @@ class Meta:
         model = RackGroup
         fields = ["id", "name", "description", "racks"]
 
+    def generate_query__ancestors(self, value):
+        """Helper method used by _ancestor() method."""
+        if value:
+            locations = Location.objects.filter(pk__in=[v.pk for v in value])
```

Comment:
Do we need to handle cases where `value` contains names instead of pks?

Reply:
`value` will always be `Location` objects I believe. Any filter field that is a `NaturalKeyOrPKMultipleChoiceFilter` with `queryset=Location.objects.all()` and a `method` attribute. The value passed in will always be the `queryset`. For example, https://github.com/nautobot/nautobot/blob/develop/nautobot/extras/filters/__init__.py#L481-L514